packages/xchain-thorchain/src/client.ts

```diff
@@ -439,7 +435,7 @@ class Client extends BaseXChainClient implements ThorchainClient, XChainClient {
       amount: amount.amount().toString(),
       asset: getDenom(asset),
       memo,
-      fee: DEFAULT_GAS_VALUE,
+      gasLimit: fee.fast,
```

Comment:
Why a change from `DEFAULT_GAS_VALUE` to `fee.fast`? `fee` is not `gasLimit`

Reply:
put it back